### ISSUE TITLE
Receiving NameError when trying to use valid?

### ISSUE THREAD

**Issue by pavanjared** [mobi/telephone_number#43]
Ruby version: `ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin16]`
OS: OSX

Here's the stack trace of the error I'm seeing:
```2.3.1 :002 > TelephoneNumber.valid?("3175082237", :US)
NameError: uninitialized constant TelephoneNumber::Number::Forwardable
	from /Users/jared.pavan/.rvm/gems/ruby-2.3.1/gems/telephone_number-1.2.0/lib/telephone_number/number.rb:3:in `<class:Number>'
	from /Users/jared.pavan/.rvm/gems/ruby-2.3.1/gems/telephone_number-1.2.0/lib/telephone_number/number.rb:2:in `<module:TelephoneNumber>'
	from /Users/jared.pavan/.rvm/gems/ruby-2.3.1/gems/telephone_number-1.2.0/lib/telephone_number/number.rb:1:in `<top (required)>'
	from /Users/jared.pavan/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/jared.pavan/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/jared.pavan/.rvm/gems/ruby-2.3.1/gems/telephone_number-1.2.0/lib/telephone_number/class_methods.rb:11:in `parse'
	from /Users/jared.pavan/.rvm/gems/ruby-2.3.1/gems/telephone_number-1.2.0/lib/telephone_number/class_methods.rb:15:in `valid?'
	from (irb):2
	from /Users/jared.pavan/.rvm/rubies/ruby-2.3.1/bin/irb:11:in `<main>'```

**Comment by pavanjared** [comment]
It seems that `forwarder` may have been removed from one of the downstream dependencies. This should be fixed by #44.

**Comment by akfernun** [comment]
Hi @pavanjared, 

Thank you for making us aware of this. I went ahead and merged #44 and cut a new release.
